src/file_pipe_log/log_file.rs

```diff
@@ -111,7 +111,10 @@ impl<F: FileSystem> LogFileWriter<F> {
 
     pub fn sync(&mut self) -> Result<()> {
         if self.last_sync < self.written {
-            let _t = StopWatch::new(&LOG_SYNC_DURATION_HISTOGRAM);
+            let _t = StopWatch::new((
+                &*LOG_SYNC_DURATION_HISTOGRAM,
+                perf_context!(log_rotate_duration),
```

Comment:
sync not rotate. And the duration here isn't equivalent to the duration of `pipe::sync`. Because rotate will sync as well (both file and directory), which should be recorded by another counter.

Reply:
I'll fix it